spec/requests/articles_spec.rb

```diff
@@ -1,61 +1,25 @@
 require "rails_helper"
 
 RSpec.describe "Articles", type: :request do
-  let(:valid_attributes) { { title: "Internship", content: "It's been a wonderful 3 months" } }
-  let(:invalid_attributes) { { title: "", content: "" } }
+  include_context "when admin signed in (controllers)"
 
-  before do
-    post user_path, params: { user: attributes_for(:user) }
-    User.last.update(role: :admin)
-  end
+  let(:valid_attributes) { { title: "Internship", content: "It's been a wonderful 3 months" } }
 
   describe "GET /index" do
-    it "renders a successful response" do
-      get articles_url
-      expect(response).to be_successful
-    end
-
     it "renders an message about no articles" do
       get articles_url
-      expect(response.body).to include("But it is not.")
-    end
-  end
-
-  describe "POST /create" do
-    context "with valid parameters" do
-      it "creates a new article" do
-        expect { post articles_path, params: { article: valid_attributes } }.to change(Article, :count).by(1)
-      end
 
-      it "redirects to the new article page" do
-        post articles_path, params: { article: valid_attributes }
-        expect(response).to redirect_to(article_path(Article.last))
-      end
-    end
-
-    context "with invalid parameters" do
-      it "does not create a new article" do
-        expect { post articles_path, params: { article: invalid_attributes } }.to change(Article, :count).by(0)
-      end
-
-      it "renders a successful response (i.e. to display the 'new' template)" do
-        post articles_path, params: { article: invalid_attributes }
-        expect(response).to be_successful
-      end
+      expect(response).to be_successful
+      expect(response.body).to include("But it is not.")
     end
   end
 
   describe "GET /show" do
-    before do
-      post articles_path, params: { article: valid_attributes }
+    it "renders an article title" do
+      post users_articles_path, params: { article: valid_attributes }
```

Comment:
What do you think if we create article in `let` instead of sending request?
```suggestion
    let(:article) { create :article, title: "Internship", content: "It's been a wonderful 3 months" }
      
    it "renders an article title" do
      get article_path(article)

      expect(response).to be_successful
      expect(response.body).to include("Internship")
    end
```